git-hub

```diff
@@ -343,46 +445,133 @@ def json_to_toml(username, repo, pr, token, open_or_closed):
     for label in list(labels_dict):
         labels = labels + label.get('name')+", "
     to_write = to_write + 'labels = "{0}" \n'.format(labels[:-2])
+
+    comment_url = response.get('comments_url')       # fetches latest activity (comment) date
+    comment_count = 0
+    most_recent = ""
+    self_comment = False
+    comment_dates = []
+    comment_content = []
+    comment_request = Request(comment_url)
+    comment_response = urlopen(comment_request)
+    comment_dict = json.loads(comment_response.read().decode('utf-8'))
+    for c in list(comment_dict):
+        comment_count = comment_count + 1
```

Comment:
No need to do this; you can measure the length of a list:

```
comments = list(comment_dict)
comment_count = len(comments)
```

Reply:
Should I have the loop to get the other stuff like the content of the comments and the dates?